docs-v2/spec.yaml

```diff
@@ -618,6 +632,28 @@ paths:
                                     descriptions: A list of sync names that you wish to pause.
                                     items:
                                         type: string
+    /sync/status:
+        get:
+            description: Get the status of specified sync(s) (for a given connection or all applicable connections if no connection is specified)
+            parameters:
+                - name: provider_config_key
+                  in: query
+                  required: true
+                  schema:
+                      type: string
+                  description: The ID of the integration you established within Nango
+                - name: syncs
+                  in: query
+                  required: true
+                  schema:
+                      type: string
+                  description: The name of the syncs you want to fetch a status for. Pass in "*" to return all syncs per the integration
+                - name: connection_id
+                  in: query
+                  required: false
+                  schema:
+                      type: string
+                  description: The ID of the connection. If omitted, all connections will be surfaced.
```

Comment:
@khaliqgant having the response specs would help IMO! In particular exposing the different possible statuses.

Reply:
Yup, was just doing that. Added at https://github.com/NangoHQ/nango/pull/1118/commits/637ae02c9a2584c854b478da29983b817281adcb#diff-34f53ca72327337e4da01de1e7520ffca06b8a6af4c3f885f3413024e7eb6ce5R657